lib/template-mixins.js

```diff
@@ -524,12 +536,15 @@ module.exports = function (options, deprecated) {
                         throw new Error('Could not find field: ' + key);
                     }
                 }
+                if (this.disableRender) {
+                    return null;
+                }
                 if (this.html) {
                     return this.html;
                 }
                 var mixin = this.mixin || 'input-text';
                 if (mixin && res.locals[mixin] && typeof res.locals[mixin] === 'function') {
-                    return res.locals[mixin]().call(Object.assign({}, res.locals, this), this.key);
+                    return res.locals[mixin]().call(Object.assign({}, res.locals), this.key);
```

Comment:
Specifically, further up this method if a field key is defined then the field properties are looked up and assigned to `this`, so if `this` is not then used to call the mixin then I'd expect that to start failing.